README.md

```diff
@@ -58,35 +78,134 @@ as a file. All jobs should have the following format:
   Expressions should not be overlapping. Set to `null` in a job definition
   to disable using predicates completely. Defaults to `default_predicates.sql`.
 
-Long-lived job definitions are stored as YAML in `config/default_jobs.yaml`,
-then converted to JSON for submission. See `run.sh` for an example of this
-workflow using `yq`.
+### Creating batch JSON
+
+The example batch above contains two separate jobs, one per table. If you want
+to add additional tables/jobs to the batch, you can manually add the
+corresponding table objects and modify the fields as
+[listed above](#field-definitions).
+
+In practice, modifying JSON is a bit of a pain, so we store long-lived
+batch and job definitions in YAML, then convert them to JSON using `yq`.
+The file `config/default_jobs.yaml` contains definitions for three common job
+batches:
+
+1. A daily batch that pulls the most recent 2 years of each critical table.
+2. A weekend batch that pulls _all_ tables and years.
+3. A test batch that pulls a subset of tables with representative situations.
+
+### Submitting via the command line
+
+Batches are submitted to the Spark Docker cluster via the command line. The
+main job submission argument is either `--json-string` or `--json-file`.
+For example, to submit the test jobs in `config/default_jobs.yaml` via
+`--json-string`, run the following command:
+
+```bash
+docker exec spark-node-master ./submit.sh \
+    --json-string "$(yq -o=json .test_jobs ./config/default_jobs.yaml)"
+```
+
+Or from a file:
+
+```bash
+yq -o=json .test_jobs ./config/default_jobs.yaml > /tmp/jobs.json
+docker exec spark-node-master ./submit.sh --json-file /tmp/jobs.json
+```
 
-### Predicates, filtering, and partitioning
+The command line interface also has multiple optional flags:
+
+- `--run-github-workflow/--no-run-github-workflow` - Run the [`test_dbt_models`](https://github.com/ccao-data/data-architecture/blob/master/.github/workflows/test_dbt_models.yaml)
+  workflow on batch completion?
+- `--run-glue-crawler/--no-run-glue-crawler` - Run the iasWorld Glue crawler
+  on batch completion?
+- `--upload-data/--no-upload-data` - Upload extracted data to the iasWorld S3
+  bucket?
+- `--upload-logs/--no-upload-logs` - Upload batch logs to AWS CloudWatch?
```

Comment:
**[Suggestion, non-blocking]** Can we include the default values for these flags?

Reply:
Done in 51f1a71!